api.md

```diff
@@ -3406,138 +3312,143 @@ SELECT pg_size_pretty(after_compression_total_bytes) as total
 
-#### Required Arguments [](indexes_relation_size-required-arguments)
+#### Required Arguments [](hypertable_size-required-arguments)
 
 |Name|Description|
 |---|---|
-| `main_table` | Identifier of hypertable to get indexes size for.|
+| `main_table` | Name of the hypertable|
 
-#### Returns [](indexes_relation_size-returns)
-|Column|Description|
-|---|---|
-|index_name|Index on hypertable|
-|total_bytes|Size of index on disk|
+#### Returns [](hypertable_size-returns)
+Total disk space used by the specified table, including all indexes and TOAST data|
 
-#### Sample Usage [](indexes_relation_size-examples)
+#### Sample Usage [](hypertable_size-examples)
+Get size information for a hypertable.
 ```sql
-SELECT * FROM indexes_relation_size('conditions');
-```
-The expected output:
-```
-              index_name              | total_bytes
---------------------------------------+-------------
- public.conditions_device_id_time_idx |  1198620672
- public.conditions_time_idx           |   487358464
-
+SELECT hypertable_size('disttable') ;
+ hypertable_size
+-----------------
+           73728
 ```
-
 ---
 
-## indexes_relation_size_pretty() [](indexes_relation_size_pretty)
+## hypertable_index_size() [](indexes_relation_size)
 
-Get sizes of indexes on a hypertable.
+Get size of an index on a hypertable.
 
-#### Required Arguments [](indexes_relation_size_pretty-required-arguments)
+#### Required Arguments [](indexes_relation_size-required-arguments)
 
 |Name|Description|
 |---|---|
-| `main_table` | Identifier of hypertable to get indexes size for.|
+| `main_table` | Name of hypertable|
 
-#### Returns [](indexes_relation_size_pretty-returns)
-|Column|Description|
-|---|---|
-|index_name|Index on hypertable|
-|total_size|Pretty output of total_bytes|
+#### Returns [](indexes_relation_size-returns)
+Returns disk space used by the index. 
 
-#### Sample Usage [](indexes_relation_size_pretty-examples)
-```sql
-SELECT * FROM indexes_relation_size_pretty('conditions');
-```
-The expected output:
-```
+#### Sample Usage [](indexes_relation_size-examples)
 
-             index_name_              | total_size
---------------------------------------+------------
- public.conditions_device_id_time_idx | 1143 MB
- public.conditions_time_idx           | 465 MB
+Get size of a specific index on a hypertable.
+
+```sql
+\d conditions_table
+                     Table "public.test_table"
+ Column |           Type           | Collation | Nullable | Default 
+--------+--------------------------+-----------+----------+---------
+ time   | timestamp with time zone |           | not null | 
+ device | integer                  |           |          | 
+ volume | integer                  |           |          | 
+Indexes:
+    "second_index" btree ("time")
+    "test_table_time_idx" btree ("time" DESC)
+    "third_index" btree ("time")
+
+select pg_size_pretty(hypertable_index_size('second_index'));
+ pg_size_pretty 
+----------------
+ 160 kB
```

Comment:
Maybe should example first without pg_size_pretty?

(Also, for consistency, I think we are capitalizing "SELECT")